lib/generators/rspec/templates/decorator_spec.rb

```diff
@@ -1,4 +1,9 @@
-require 'rails_helper'
+<% require 'rspec/version' %>
+<% if RSpec::Version::STRING.match(/\A3\.[^10]/) %>
+  require 'rails_helper'
```

Comment:
What rspec versions are you trying to test for here with 3 not followed by a 1 or a 0?  Are you trying to do rspec version greater than 3.2?  Your regexp will fail if rspec gets to 3.10, which it's not far from. 

You may be able to use this instead?

    if Gem::Version.new('RSpec::Version::String) >= Gem::Version.new("3.2")

Reply:
Yeah, the code you have provided is a lot less brittle. This work was originally done by someone else, I just fixed the conflicts and re-opened the pull request. I can only assume they were trying to achieve what you stated above. I haven't had a lot of time lately to work on this. If you are interested in cleaning this up and getting it pushed through, you are more than welcome to take it over.

Reply:
Hmm, I think the highest priority is actually improving the template for the decorator specs that Draper generates -- this file we're commenting on now. 

The rest of the PR seems to be about changing Draper's own specs -- but if Draper's specs pass, that doesn't seem to be as much of a priority, as it doesn't really effect the developer _users_ of Draper, like the Draper generator for specs in your own app does. 

So if I can find the time, I might open another PR just focusing on that? It sounds like you are a Draper committer/maintainer, so I have some sense that if I can find time to do that, it'll have some chance of being reviewed/merged and my work won't be in vain? :) Thanks!

Reply:
I think that makes perfect sense. I'd love to see that get improved. And yes, I  will be able to review and merge. Sometimes I'm slower to review that I'd personally like, so if that becomes the case, just bug me. Thanks for the help!